src/server/journal/streamer.cc

```diff
@@ -362,4 +362,8 @@ void RestoreStreamer::WriteEntry(string_view key, const PrimeValue& pk, const Pr
   stats_.commands += serializer.SerializeEntry(key, pk, pv, expire_ms);
 }
 
+size_t RestoreStreamer::KeysWrittenStats() const {
+  return stats_.keys_written.load(std::memory_order_relaxed);
```

Comment:
I do not think it's justified to make an arbitrary variable atomic, nor I understand the reason for this.
Why we suddenly allow an access to streamer object from another thread? where do we need it? will AwaitBrief solve the issue?